metadata-protobuf/proto/Membership.proto

```diff
@@ -25,5 +25,11 @@ message MembershipMetadata {
     optional ResourceType type = 1;
     optional string value = 2;
   }
+  
   repeated ExternalResource externalResources = 5;
+  
+  optional bool isVerifiedValidator = 6;
```

Comment:
I just realized there shouldn't be any `isVerifiedValidator` here. The `MembershipMetadata` can only define a `validatorAccount` but not whether or not it's verified.